http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -143,15 +141,18 @@ class LeoPubsubMessageSubscriber[F[_]: Async: Timer: ContextShift: Logger: Concu
 }
 
 sealed trait LeoPubsubMessage {
+  def traceId: Option[TraceId]
   def messageType: String
 }
 
 object LeoPubsubMessage {
-  final case class StopUpdateMessage(updatedMachineConfig: MachineConfig, clusterId: Long) extends LeoPubsubMessage {
+  final case class StopUpdateMessage(updatedMachineConfig: MachineConfig, clusterId: Long, traceId: Option[TraceId])
+      extends LeoPubsubMessage {
     val messageType = "stopUpdate"
   }
 
-  case class ClusterTransitionFinishedMessage(clusterFollowupDetails: ClusterFollowupDetails) extends LeoPubsubMessage {
+  case class ClusterTransitionFinishedMessage(clusterFollowupDetails: ClusterFollowupDetails, traceId: Option[TraceId])
```

Comment:
can we give this a default value of None

Reply:
I kinda prefer not having a default `None`. Because that way, people tend to ignore the argument, but I really want to encourage people to use it.